security/monitoring.adoc

```diff
@@ -89,7 +89,7 @@ This section describes the types of operational logs produced on the cluster.
 [[security-monitoring-service-logs]]
 === Service Logs
 
-{product-title} produces logs with each *systemd* service that is running on a host:
+{product-title} produces logs for services that run on static pods in a cluster:
```

Comment:
Perhaps you also want to mention that apiserver, controllers and etcd static pods run in kube-system namespace.